### PR TITLE
adds span's so that pagebrowser can be formatted like a button bar

### PR DESCRIPTION
**Pull request by BFallert**
With the added spans its possible to format the PageBrowser like GitHub's PageBrowser 
image

### CODE REVIEW

File: Classes/Plugin/PageGrid.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ protected function getPageBrowser() {
         if ($this->piVars['pointer'] > 0) {
             $output = $this->pi_linkTP_keepPIvars($this->pi_getLL('prevPage', '&lt;', TRUE), ['pointer' => $this->piVars['pointer'] - 1, 'page' => (($this->piVars['pointer'] - 1) * $this->conf['limit']) + 1], TRUE).$separator;
         } else {
-            $output = $this->pi_getLL('prevPage', '&lt;', TRUE).$separator;
+            $output = '<span class="first-page not-active">'.$this->pi_getLL('prevPage', '&lt;', TRUE).'</span>'.$separator;
```
Comment (sebastian-meyer) [inline Classes/Plugin/PageGrid.php]: Shouldn't this be something like `prev-page` instead of `first-page`?

File: Classes/Plugin/PageGrid.php [hunk 2]
```diff
@@ -100,7 +100,7 @@ protected function getPageBrowser() {
                 if ($this->piVars['pointer'] != $i) {
                     $output .= $this->pi_linkTP_keepPIvars(sprintf($this->pi_getLL('page', '%d', TRUE), $i + 1), ['pointer' => $i, 'page' => ($i * $this->conf['limit']) + 1], TRUE).$separator;
                 } else {
-                    $output .= sprintf($this->pi_getLL('page', '%d', TRUE), $i + 1).$separator;
+                    $output .= '<span class="act-page">'.sprintf($this->pi_getLL('page', '%d', TRUE), $i + 1).'</span>'.$separator;
```
Comment (sebastian-meyer) [inline Classes/Plugin/PageGrid.php]: I suggest using `active` here instead of `act-page` since that's the opposite of `not-active`.

**Review by sebastian-meyer — CHANGES_REQUESTED** [review]
Just some minor wording suggestions.

File: Classes/Plugin/PageGrid.php [hunk 3]
```diff
@@ -100,11 +100,11 @@ protected function getPageBrowser() {
...
                 }
                 $skip = TRUE;
             } elseif ($skip == TRUE) {
-                $output .= $this->pi_getLL('skip', '...', TRUE).$separator;
+                $output .= '<span class="skipped">'.$this->pi_getLL('skip', '...', TRUE).'</span>'.$separator;
```
Comment (sebastian-meyer) [inline Classes/Plugin/PageGrid.php:107]: Added a `<span>` for skipped pages as well.

File: Classes/Plugin/PageGrid.php (hunk 1 above)
Comment (BFallert) [reply Classes/Plugin/PageGrid.php]: you are right, my mistake! 
```suggestion
            $output = '<span class="prev-page not-active">'.$this->pi_getLL('prevPage', '&lt;', TRUE).'</span>'.$separator;
```

File: Classes/Plugin/PageGrid.php (hunk 2 above)
Comment (BFallert) [reply Classes/Plugin/PageGrid.php]: ok, you have more experience
```suggestion
                    $output .= '<span class="active">'.sprintf($this->pi_getLL('page', '%d', TRUE), $i + 1).'</span>'.$separator;
```

**Comment by BFallert** [comment]
I agree with all the adjustments

**Review by sebastian-meyer — APPROVED** [review]
Thank you very much!
(Sometimes I am a fusspot.)

**Pull request merged**